packages/models/src/lib/implementation/utils.ts

```diff
@@ -108,10 +108,10 @@ export function getMissingRefsForCategories(
 }
 
 export function missingRefsForCategoriesErrorMsg(
-  categories: CategoryConfig[],
   plugins: PluginConfig[] | PluginReport[],
+  categories?: CategoryConfig[],
 ) {
-  const missingRefs = getMissingRefsForCategories(categories, plugins);
+  const missingRefs = getMissingRefsForCategories(plugins, categories);
```

Comment:
The parameters are quite unintuitive now. Both functions are primarily with categories (going by their name), yet the categories are an optional second parameter.

I would expect `categories: CategoryConfig[] | undefined, plugins: PluginConfig[] | PluginReport[]`.  It would be a mistake to call this function without passing in anything for categories, so `| undefined` is more appropriate than `?`.  And since categories are most relevant, they should be specified first, the plugins are more like additional context here.